content/applications/finance/payment_acquirers.rst

```diff
@@ -108,11 +109,11 @@ Online payment acquirers
      -
      -
      -
-   * - PayUMoney
```

Comment:
Checking with a fresh trial database, PayUmoney remains the payment acquirer and I cannot find Razorpay. I guess this is for an upcoming change?

Reply:
@toaa-odoo , Yes it's for upcoming changes. Thanks